ext/civicrm_admin_ui/managed/SavedSearch_Contact_Summary_Membership_Type.mgd.php

```diff
@@ -0,0 +1,154 @@
+        'label' => E::ts('Contact Summary Membership Type'),
+        'saved_search_id.name' => 'Contact_Summary_Membership_Type',
+        'type' => 'table',
+        'settings' => [
+          'description' => E::ts('The following Membership Types are associated with this organization. Click Members for a listing of all contacts who have memberships of that type. Click Edit to modify the settings for that type.'),
+          'sort' => [],
+          'limit' => 50,
+          'pager' => [
+            'hide_single' => TRUE,
+          ],
+          'placeholder' => 5,
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'name',
+              'dataType' => 'String',
+              'label' => E::ts('Name'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'period_type:label',
+              'dataType' => 'String',
+              'label' => E::ts('Period'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'fixed_period_start_day',
+              'dataType' => 'Integer',
+              'label' => E::ts('Fixed Start'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'minimum_fee',
+              'dataType' => 'Money',
+              'label' => E::ts('Minimum Fee'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'duration_interval',
+              'dataType' => 'Integer',
+              'label' => E::ts('Duration'),
+              'sortable' => TRUE,
+              'rewrite' => '[duration_interval] [duration_unit:label]',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'visibility:label',
+              'dataType' => 'String',
+              'label' => E::ts('Visibility'),
+              'sortable' => TRUE,
+            ],
+            [
+              'size' => 'btn-xs',
+              'links' => [
+                [
+                  'path' => 'civicrm/member/search?reset=1&force=1&type=[id]',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Members'),
+                  'style' => 'default',
+                  'condition' => [],
+                  'task' => '',
+                  'entity' => '',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                ],
+                [
+                  'task' => '',
+                  'entity' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Edit'),
+                  'style' => 'default',
+                  'path' => 'civicrm/admin/member/membershipType/add?action=update&id=[id]&reset=1',
```

Comment:
Can this use entity+action instead of path?

Reply:
Have now added links for MembershipType entity in xml & updated the same here. Thanks.